src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import styles from "./Pagination.module.scss";
+
+export const Pagination = ({ postsPerPage, totalPosts, paginate }) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalPosts / postsPerPage); i++) {
```

Comment:
You could move it to a helper or even utils functions and just import and use it here when you need it.